### PR TITLE
Follow up on use case formats

### PR DESCRIPTION
**Pull request by jwitz**
This PR attempts to hone the purpose of the new Astro "use case" section by simplifying the content of each document. When reading through the current doc, I noticed some potential improvements that could be made to the format:

- The current doc is long, and Astro customers might be turned off from seeing a multitude of steps and explanations for something we want to promote as a simplified solution.
- The example project only works with select technologies. For Astro customers that are more locked in to specific clouds and services, seeing a mention of S3 might turn them off from reading the use case altogether.
- The current titles for steps are really similar to base Astro documentation, and I could see users getting confused about the purpose of this doc when these steps also exist elsewhere in documentation.

This new format relies more on linking out to other docs, but has more dedicated content explaining the benefit of the use case.

### CODE REVIEW

File: astro/use-cases/connections-branch-deploys.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- Configuring Airflow connections in the [Astro Environment Manager](https://docs.astronomer.io/astro/manage-connections-variables).
+- Branch-based, or multi-branch Deployments using [CI/CD](https://docs.astronomer.io/astro/set-up-ci-cd.md#multiple-environments).
+
+## Prerequisites
+
+This use case assumes you have:
+
+- At least two Astro Deployments, one for development and one for production.
+- A Git-based repository where you manage an Astro project.
+- A configured multi-branch CI/CD pipeline. See [CI/CD templates](ci-cd-templates.md)
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:30]:
```suggestion
- A configured multi-branch CI/CD pipeline. See [CI/CD templates](ci-cd-templates/overview.md)
```

File: astro/use-cases/connections-branch-deploys.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Configuring Airflow connections in the [Astro Environment Manager](https://docs.astronomer.io/astro/manage-connections-variables).
+- Branch-based, or multi-branch Deployments using [CI/CD](https://docs.astronomer.io/astro/set-up-ci-cd.md#multiple-environments).
+
+## Prerequisites
+
+This use case assumes you have:
+
+- At least two Astro Deployments, one for development and one for production.
+- A Git-based repository where you manage an Astro project.
+- A configured multi-branch CI/CD pipeline. See [CI/CD templates](ci-cd-templates/overview.md)
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:30]:
```suggestion
- A configured multi-branch CI/CD pipeline. See [CI/CD templates](ci-cd-templates/template-overview.md)
```

File: astro/use-cases/connections-branch-deploys.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Implementation
+
+To implement this use case:
+
+1. As a Workspace Operator or Admin, create connections in the Astro Environment manager to your development-level resources. See [Create a connection](create-and-link-connections.md#create-a-connection).
+2. Set these connections to be available to all Deployments in the Workspace by default. See [Configure connection sharing for a Workspace](create-and-link-connections.md#configure-connection-sharing-for-a-workspace). Your DAG authors can now access external development and testing resources across all Deployments.
+3. Choose a Deployment in your Workspace that you want to run your production workflows. In the Deployment, override the connections you configured to instead connect to your production resources. See [Override connection fields](create-and-link-connections.md#override-connection-fields).
+4. In the Git repository for your Astro project, define a multi-branch CI/CD pipeline for deploying to Astro. For an example of how to do this in GitHub, see [CI/CD templates](/ci-cd-temploates/github-actions.md?tab=multibranch#deploy-action-templates).
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:41]:
```suggestion
4. In the Git repository for your Astro project, define a multi-branch CI/CD pipeline for deploying to Astro. For an example of how to do this in GitHub, see [CI/CD templates](/ci-cd-templates/github-actions.md?tab=multibranch#deploy-action-templates).
```

File: astro/use-cases/connections-branch-deploys.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+After a Workspace Owner creates connections, DAG authors can develop DAGs without needing to reconfigure connections between development and production:
+
+- A DAG author creates a new development branch of an Astro project. The CI/CD pipeline for the repository deploys their branch to the development Deployment on Astro. This is known as a [multi-branch CI/CD pipeline](https://docs.astronomer.io/astro/set-up-ci-cd#multiple-environments).
+- While the DAG author is testing in Astro, they have access to the development resources because the Workspace Author configured a default Airflow connection for the Deployment. The author can [pull this connection onto their local machine](https://docs.astronomer.io/astro/import-export-connections-variables.md#from-the-cloud-ui) for testing purposes, or test by deploying to the development Deployment on Astro.
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:55]:
```suggestion
- While the DAG author is testing in Astro, they have access to the development resources because the Workspace Author configured a default Airflow connection for the Deployment. The author can [pull this connection onto their local machine](import-export-connections-variables.md#from-the-cloud-ui) for testing purposes, or test by deploying to the development Deployment on Astro.
```

File: astro/use-cases/connections-branch-deploys.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+- In the production Deployment, the Workspace Owner [overrides the linked connection](https://docs.astronomer.io/astro/create-and-link-connections#override-connection-fields) to instead connect to production resources on the same external system. Because the connection ID and code are the same, this override requires no updates at the DAG level. 
+
+After a Workspace Owner creates connections, DAG authors can develop DAGs without needing to reconfigure connections between development and production:
+
+- A DAG author creates a new development branch of an Astro project. The CI/CD pipeline for the repository deploys their branch to the development Deployment on Astro. This is known as a [multi-branch CI/CD pipeline](https://docs.astronomer.io/astro/set-up-ci-cd#multiple-environments).
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:54]:
```suggestion
- A DAG author creates a new development branch of an Astro project. The CI/CD pipeline for the repository deploys their branch to the development Deployment on Astro. This is known as a [multi-branch CI/CD pipeline](set-up-ci-cd.md#multiple-environments).
```

File: astro/use-cases/connections-branch-deploys.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+## Explanation
+
+Using branch-based Deployments with the Astro Environment Manager allows your team to focus on the parts of Astro that matter most for their roles. For example, using the Astro Environment Manager means that you only need one administrative user to manage connections across multiple Deployments:  
+
+- A Workspace Owner [creates an Airflow connection](https://docs.astronomer.io/astro/create-and-link-connections#create-a-connection) in the Astro Environment Manager that connects to external development resources. They share this connection to all Deployments by default by turning on the [**Linked to all Deployments** setting](https://docs.astronomer.io/astro/create-and-link-connections#configure-connection-sharing-for-a-workspace).
+- In the production Deployment, the Workspace Owner [overrides the linked connection](https://docs.astronomer.io/astro/create-and-link-connections#override-connection-fields) to instead connect to production resources on the same external system. Because the connection ID and code are the same, this override requires no updates at the DAG level.
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:50]:
```suggestion
- A Workspace Owner [creates an Airflow connection](https://docs.astronomer.io/astro/create-and-link-connections#create-a-connection) in the Astro Environment Manager that connects to external development resources. They share this connection to all Deployments by default by turning on the [**Linked to all Deployments** setting](https://docs.astronomer.io/astro/create-and-link-connections#configure-connection-sharing-for-a-workspace).
- In the production Deployment, the Workspace Owner [overrides the linked connection](create-and-link-connections.md#override-connection-fields) to instead connect to production resources on the same external system. Because the connection ID and code are the same, this override requires no updates at the DAG level. 
```

File: astro/use-cases/connections-branch-deploys.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+This use case provides several benefits for both Workspace managers and DAG authors:
+
+- Workspace Operators and Owners can manage connections without needing access to DAG code.
+- DAG authors only need a connection ID to connect their DAGs, meaning they can focus on data engineering instead of connection configuration.
+- Connection IDs don't need to be updated when you promote code to Deployment, reducing development timelines and reducing the number of resources to manage. 
+
+## See also
+
+- [Manage Airflow connections and variables](https://docs.astronomer.io/astro/manage-connections-variables)
+- [Automate actions on Astro](https://docs.astronomer.io/astro/automation-overview)
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:68]:
```suggestion
- [Manage Airflow connections and variables](manage-connections-variables.md)
- [Automate actions on Astro](automation-overview.md)
```

File: astro/use-cases/connections-branch-deploys.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+- Automatically create Deployments for development branches with a set of development Airflow connections.
+- Override preset Airflow connections on a per-Deployment basis to troubleshoot or customize your development environment.
+- Promote a Deployment to production as an admin by updating its Airflow connections to access production resources.
+
+## Feature overview
+
+This use case depends on three key Astro features to create a fully integrated CI/CD pipeline:
+
+- Configuring Airflow connections in the [Astro Environment Manager](https://docs.astronomer.io/astro/manage-connections-variables).
+- Branch-based, or multi-branch Deployments using [CI/CD](https://docs.astronomer.io/astro/set-up-ci-cd.md#multiple-environments).
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:22]:
```suggestion
- Configuring Airflow connections in the [Astro Environment Manager](manage-connections-variables.md).
- Branch-based, or multi-branch Deployments using [CI/CD](set-up-ci-cd.md#multiple-environments).
```

File: astro/use-cases/connections-branch-deploys.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
+---
+title: 'Manage Astro connections in branch-based deploy workflows'
+sidebar_label: 'Connections + branch-based deploys'
+id: connections-branch-deploys
+---
+
+Airflow DAGs often interact with a multitude of external systems, such as data warehouses and APIs. DAGs access these systems using [Airflow connections](https://docs.astronomer.io/astro/manage-connections-variables). A common logistical consideration when running Airflow at scale is deciding how to manage connections between development and production environments. Different environment types require different levels access to external resources. 
+
+Astro's [branch-based development](https://docs.astronomer.io/astro/automation-overview) and [connection management](https://docs.astronomer.io/astro/manage-connections-variables) features allow you to automatically share specific Airflow connections with Astro Deployments based on their development context.
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:9]:
```suggestion
Airflow DAGs often interact with a multitude of external systems, such as data warehouses and APIs. DAGs access these systems using [Airflow connections](manage-connections-variables.md). A common logistical consideration when running Airflow at scale is deciding how to manage connections between development and production environments. Different environment types require different levels access to external resources. 

Astro's [branch-based development](automation-overview.md) and [connection management](manage-connections-variables.md) features allow you to automatically share specific Airflow connections with Astro Deployments based on their development context. 
```

File: astro/use-cases/connections-branch-deploys.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+
+When you combine branch-based deploys and connection management, you can:
+
+- Automatically create Deployments for development branches with a set of development Airflow connections.
+- Override preset Airflow connections on a per-Deployment basis to troubleshoot or customize your development environment.
+- Promote a Deployment to production as an admin by updating its Airflow connections to access production resources.
+
+## Feature overview
+
+This use case depends on three key Astro features to create a fully integrated CI/CD pipeline:
```
Comment (lzdanski) [inline astro/use-cases/connections-branch-deploys.md:19]: this says three features, but only lists two

File: astro/use-cases/connections-branch-deploys.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+3. Choose a Deployment in your Workspace that you want to run your production workflows. In the Deployment, override the connections you configured to instead connect to your production resources. See [Override connection fields](create-and-link-connections.md#override-connection-fields).
+4. In the Git repository for your Astro project, define a multi-branch CI/CD pipeline for deploying to Astro. For an example of how to do this in GitHub, see [CI/CD templates](/ci-cd-templates/github-actions.md?tab=multibranch#deploy-action-templates).
+
+Now, when a DAG author deploys to either a production or development Deployment through CI/CD, they can run their DAGs in Astro without needing to configure connections. Additionally, because your override production connection shares the same connection ID as your default development connection, DAG authors don't have to update their code to point towards different connections when promoting their code to production.
```
Comment (lzdanski) [inline astro/use-cases/connections-branch-deploys.md:43]:
```suggestion
Now, when a DAG author deploys to either a production or development Deployment through CI/CD, they can run their DAGs in Astro without needing to configure connections. Additionally, because your production connection shares the same connection ID as your default development connection, DAG authors don't have to update their code to point towards different connections when promoting their code to production.
```

File: astro/use-cases/connections-branch-deploys.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+## Explanation
+
+Using branch-based Deployments with the Astro Environment Manager allows your team to focus on the parts of Astro that matter most for their roles. For example, using the Astro Environment Manager means that you only need one administrative user to manage connections across multiple Deployments:
```
Comment (lzdanski) [inline astro/use-cases/connections-branch-deploys.md:47]: LOVE this section

**Review by lzdanski — COMMENTED** [review]
Just some small comments! I really like this version and the addition of the numbered implementation steps really helps synthesize everything and there's such explicit value statements.

File: astro/use-cases/connections-branch-deploys.md (hunk 10 above)
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:19]:
```suggestion
This use case depends on the following Astro features to create a fully integrated CI/CD pipeline:
```

**Comment by jwitz** [comment]
Hey @kentdanas and @TJaniF I was working through the use case last week and had some ideas for how we could simplify the content for Astro customers. The original format works great for OSS tutorials, but I found it hard to parse from the perspective of a customer trying to figure out the "why" of how these features work together. 

This rewrite tries to focus more on the concept of the use case than the implementation. I know I cut a lot of content (sorry Tamara 😬 ) so please push back if you think there's anything that's missing, or if you think there's a way we can still include the example repo. Thank you!

File: astro/use-cases/connections-branch-deploys.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
+---
+title: 'Manage Astro connections in branch-based deploy workflows'
+sidebar_label: 'Connections and branch-based deploys'
+id: connections-branch-deploys
+---
+
+Airflow DAGs often interact with a multitude of external systems, such as data warehouses and APIs. DAGs access these systems using [Airflow connections](manage-connections-variables.md). A common logistical consideration when running Airflow at scale is deciding how to manage connections between development and production environments. Different environment types require different levels access to external resources.
```
Comment (TJaniF) [inline astro/use-cases/connections-branch-deploys.md:7]:
```suggestion
Airflow DAGs often interact with a multitude of external systems, such as data warehouses and APIs. DAGs access these systems using [Airflow connections](manage-connections-variables.md). A common logistical consideration when running Airflow at scale is deciding how to manage connections between development and production environments. Different environment types require different levels of access to external resources. 
```

File: astro/use-cases/connections-branch-deploys.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+- Configuring Airflow connections in the [Astro Environment Manager](manage-connections-variables.md).
+- Branch-based, or multi-branch Deployments using [CI/CD](set-up-ci-cd.md#multiple-environments).
+
+## Prerequisites
+
+This use case assumes you have:
+
+- At least two Astro Deployments, one for development and one for production.
+- A Git-based repository where you manage an Astro project.
+- A configured multi-branch CI/CD pipeline. See [CI/CD templates](ci-cd-templates/template-overview.md)
```
Comment (TJaniF) [inline astro/use-cases/connections-branch-deploys.md:30]:
```suggestion
- A configured multi-branch CI/CD pipeline. See [CI/CD templates](ci-cd-templates/template-overview.md).
```

File: astro/use-cases/connections-branch-deploys.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+- In the production Deployment, the Workspace Owner [overrides the linked connection](create-and-link-connections.md#override-connection-fields) to instead connect to production resources on the same external system. Because the connection ID and code are the same, this override requires no updates at the DAG level. 
+
+After a Workspace Owner creates connections, DAG authors can develop DAGs without needing to reconfigure connections between development and production:
+
+- A DAG author creates a new development branch of an Astro project. The CI/CD pipeline for the repository deploys their branch to the development Deployment on Astro. This is known as a [multi-branch CI/CD pipeline](set-up-ci-cd.md#multiple-environments).
+- While the DAG author is testing in Astro, they have access to the development resources because the Workspace Author configured a default Airflow connection for the Deployment. The author can [pull this connection onto their local machine](import-export-connections-variables.md#from-the-cloud-ui) for testing purposes, or test by deploying to the development Deployment on Astro.
```
Comment (TJaniF) [inline astro/use-cases/connections-branch-deploys.md:55]: This confused me slightly because `Astro` refers to the cloud to me. Maybe `in Astro or a local development environment` ?

File: astro/use-cases/connections-branch-deploys.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+- When the DAG author finishes development, they merge their development branch into production. The CI/CD pipeline deploys this change to the production Deployment.
+- When the DAG author's code runs in the production Deployment, it now accesses production resources based on the overrides configured by the Workspace Owner. 
+
+This use case provides several benefits for both Workspace managers and DAG authors:
+
+- Workspace Operators and Owners can manage connections without needing access to DAG code.
+- DAG authors only need a connection ID to connect their DAGs, meaning they can focus on data engineering instead of connection configuration.
+- Connection IDs don't need to be updated when you promote code to Deployment, reducing development timelines and reducing the number of resources to manage.
```
Comment (TJaniF) [inline astro/use-cases/connections-branch-deploys.md:63]:
```suggestion
- Connection IDs don't need to be updated when you promote code to production, reducing development timelines and reducing the number of resources to manage. 
```

**Review by TJaniF — COMMENTED** [review]
Aww a little sad about the 🍪 s but I think you are right, this is much more to the point. Left a couple of comments and will discuss with @kentdanas later today :)

File: astro/use-cases/connections-branch-deploys.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+
+Airflow DAGs often interact with a multitude of external systems, such as data warehouses and APIs. DAGs access these systems using [Airflow connections](manage-connections-variables.md). A common logistical consideration when running Airflow at scale is deciding how to manage connections between development and production environments. Different environment types require different levels access to external resources. 
+
+Astro's [branch-based development](automation-overview.md) and [connection management](manage-connections-variables.md) features allow you to automatically share specific Airflow connections with Astro Deployments based on their development context. 
+
+When you combine branch-based deploys and connection management, you can:
+
+- Automatically create Deployments for development branches with a set of development Airflow connections.
+- Override preset Airflow connections on a per-Deployment basis to troubleshoot or customize your development environment.
+- Promote a Deployment to production as an admin by updating its Airflow connections to access production resources.
```
Comment (kentdanas) [inline astro/use-cases/connections-branch-deploys.md:15]: I find "promote a deployment to production" confusing, because to me that implies something is changing with the deployment itself and that there is some in-product "production" designation. Could we instead say something like "Manage production Deployments without code changes by updating its Airflow connections to access production resources"?

File: astro/use-cases/connections-branch-deploys.md [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+
+- A DAG author creates a new development branch of an Astro project. The CI/CD pipeline for the repository deploys their branch to the development Deployment on Astro. This is known as a [multi-branch CI/CD pipeline](set-up-ci-cd.md#multiple-environments).
+- While the DAG author is testing in Astro, they have access to the development resources because the Workspace Author configured a default Airflow connection for the Deployment. The author can [pull this connection onto their local machine](import-export-connections-variables.md#from-the-cloud-ui) for testing purposes, or test by deploying to the development Deployment on Astro.
+- When the DAG author finishes development, they merge their development branch into production. The CI/CD pipeline deploys this change to the production Deployment.
+- When the DAG author's code runs in the production Deployment, it now accesses production resources based on the overrides configured by the Workspace Owner. 
+
+This use case provides several benefits for both Workspace managers and DAG authors:
```
Comment (kentdanas) [inline astro/use-cases/connections-branch-deploys.md:59]: I would add a bullet here for the benefits of branch-based deploys, something like your CICD pipeline manages the lifecycle of development Deployments, meaning you can save resources, etc.

**Review by kentdanas — APPROVED** [review]
@jwitz this is a great improvement, thanks for this. I left a few minor suggestions. My only other comment, which is more wish-list than requirement, is that since we've pared this back, it might be helpful to have a diagram that shows the CICD flow and something to indicate pulling in connections at different steps, similar to what we have in our CICD docs.

File: astro/use-cases/connections-branch-deploys.md [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+
+Airflow DAGs often interact with a multitude of external systems, such as data warehouses and APIs. DAGs access these systems using [Airflow connections](manage-connections-variables.md). A common logistical consideration when running Airflow at scale is deciding how to manage connections between development and production environments. Different environment types require different levels of access to external resources. 
+
+Astro's [branch-based development](automation-overview.md) and [connection management](manage-connections-variables.md) features allow you to automatically share specific Airflow connections with Astro Deployments based on their development context. 
+
+When you combine branch-based deploys and connection management, you can:
+
+- Automatically create Deployments for development branches with a set of development Airflow connections.
+- Override preset Airflow connections on a per-Deployment basis to troubleshoot or customize your development environment.
+- Promote a Deployment to production as an admin by updating its Airflow connections to access production resources.
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:15]:
```suggestion
- Run a development and production Deployment for the same project without changing Airflow connection names between the two contexts
```

File: astro/use-cases/connections-branch-deploys.md [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+
+- A DAG author creates a new development branch of an Astro project. The CI/CD pipeline for the repository deploys their branch to the development Deployment on Astro. This is known as a [multi-branch CI/CD pipeline](set-up-ci-cd.md#multiple-environments).
+- While the DAG author is testing in Astro, they have access to the development resources because the Workspace Author configured a default Airflow connection for the Deployment. The author can [pull this connection onto their local machine](import-export-connections-variables.md#from-the-cloud-ui) for testing purposes, or test by deploying to the development Deployment on Astro.
+- When the DAG author finishes development, they merge their development branch into production. The CI/CD pipeline deploys this change to the production Deployment.
+- When the DAG author's code runs in the production Deployment, it now accesses production resources based on the overrides configured by the Workspace Owner. 
+
+This use case provides several benefits for both Workspace managers and DAG authors:
+
+- Workspace Operators and Owners can manage connections without needing access to DAG code.
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:61]:
```suggestion
- When you use branch-based deploys, your CI/CD pipeline automatically deploys your branches to development Deployments. This saves resources and reduces the complexity of development for DAG authors.
- Workspace Operators and Owners can manage connections without needing access to DAG code.
```

File: astro/use-cases/connections-branch-deploys.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+- In the production Deployment, the Workspace Owner [overrides the linked connection](create-and-link-connections.md#override-connection-fields) to instead connect to production resources on the same external system. Because the connection ID and code are the same, this override requires no updates at the DAG level. 
+
+After a Workspace Owner creates connections, DAG authors can develop DAGs without needing to reconfigure connections between development and production:
+
+- A DAG author creates a new development branch of an Astro project. The CI/CD pipeline for the repository deploys their branch to the development Deployment on Astro. This is known as a [multi-branch CI/CD pipeline](set-up-ci-cd.md#multiple-environments).
+- While the DAG author is testing in Astro, they have access to the development resources because the Workspace Author configured a default Airflow connection for the Deployment. The author can [pull this connection onto their local machine](import-export-connections-variables.md#from-the-cloud-ui) for testing purposes, or test by deploying to the development Deployment on Astro.
```
Comment (jwitz) [inline astro/use-cases/connections-branch-deploys.md:55]:
```suggestion
- The DAG author has access to development resources because the Workspace Author configured a default Airflow connection for the Deployment. The author can [pull this connection onto their local machine](import-export-connections-variables.md#from-the-cloud-ui) for testing purposes, or test by deploying to the development Deployment on Astro.
```

**Comment by jwitz** [comment]
@lzdanski Added a few more bullet points, after that I think we're good to merge!

File: astro/manage-connections-variables.md [hunk 22]
```diff
@@ -77,7 +77,7 @@ Astro includes connection management system that behaves like you are using an A
 
 :::info
 
-To see how you can use connections set in the Astro Cloud UI Environment Manager in a best practice branch-based Deployment set up, see the [Manage Astro connections in branch-based deploy workflows](astro-use-case/use-case-astro-connections) use case.
+To see how you can use connections set in the Astro Cloud UI Environment Manager in a best practice, branch-based Deployment setup, see the [Manage Astro connections in branch-based deploy workflows](use-cases/connections-branch-deploys.md) use case.
```
Comment (lzdanski) [inline astro/manage-connections-variables.md:80]: I think you might also need a redirect. The original use case doc was promoted pretty heavily in a lot of marketing/outreach materials. We don't want that to break!

File: astro/use-cases/connections-branch-deploys.md [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+
+Astro's [branch-based development](automation-overview.md) and [connection management](manage-connections-variables.md) features allow you to automatically share specific Airflow connections with Astro Deployments based on their development context. 
+
+When you combine branch-based deploys and connection management, you can:
+
+- Automatically create Deployments for development branches with a set of development Airflow connections.
+- Override preset Airflow connections on a per-Deployment basis to troubleshoot or customize your development environment.
+- Run a development and production Deployment for the same project without changing Airflow connection names between the two contexts.
+
+## Feature overview
```
Comment (lzdanski) [inline astro/use-cases/connections-branch-deploys.md:17]: I really love these headings as a standard for use cases. It cuts right to the chase and the essentials of what's being used, what's required, how to combine them, and what's happening. Big fan.

File: astro/use-cases/connections-branch-deploys.md [hunk 24]
```diff
@@ -0,0 +1,69 @@
...
+4. In the Git repository for your Astro project, define a multi-branch CI/CD pipeline for deploying to Astro. For an example of how to do this in GitHub, see [CI/CD templates](/ci-cd-templates/github-actions.md?tab=multibranch#deploy-action-templates).
+
+Now, when a DAG author deploys to either a production or development Deployment through CI/CD, they can run their DAGs in Astro without needing to configure connections. Additionally, because your production connection shares the same connection ID as your default development connection, DAG authors don't have to update their code to point towards different connections when promoting their code to production.
+
+## Explanation
+
+Using branch-based Deployments with the Astro Environment Manager allows your team to focus on the parts of Astro that matter most for their roles. For example, using the Astro Environment Manager means that you only need one administrative user to manage connections across multiple Deployments:
```
Comment (lzdanski) [inline astro/use-cases/connections-branch-deploys.md:47]: we've definitely hit the point that we need to have some dedicated Environment Manager docs docs. Will probably need to go into the Astro overview/architecture page eventually. This use case has a lot of excellent summary about it.

**Review by lzdanski — APPROVED** [review]
Do we need a redirect with the new file name/ clean sweep of all instances with the old one? Other than that, read through on the preview and it looks great.

**Comment by jwitz** [comment]
Confirmed redirect works, merging!
